client/src/planwise/client/projects2/components/dashboard.cljs

```diff
@@ -69,36 +78,38 @@
     :else nil))
 
 (defn- scenarios-sortable-header
-  [{:keys [sortable] :as props} title field]
+  [{:keys [sortable] :as props} field title tooltip]
```

Comment:
I'd also send all properties as the first map argument. Eg. use it as
```clojure
[scenarios-sortable-header
  {:class       [:col-name]
   :align       :left
   :sorting-key :name 
   :tooltip     "The name of the scenario"}
  "Name"]
```
You can `dissoc` the extra keys (not appropriate for `ui/sortable-table-header`) if necessary.

Reply:
Using props the code is cleaner.